test/local/db_tests.js

```diff
@@ -217,7 +218,8 @@ test(
       name: '',
       type: 'mobile',
       pushCallback: 'https://foo/bar',
-      pushPublicKey: crypto.randomBytes(32)
+      pushPublicKey: base64url(crypto.randomBytes(65)),
```

Comment:
Heh, want to be more accurate? `base64url("\x04" + crypto.randomBytes(64))`
Might be a good idea to do that just in case the Push Server starts doing up front validation checks.